services/community-site/src/services/useDposSubgraph.ts

```diff
@@ -0,0 +1,46 @@
+import { useMemo } from 'react';
+import useApi from './useApi';
+import { DelegationGQL } from '../interfaces/Delegation';
+import { CommissionChangeGQL } from '../interfaces/Validator';
+
+export default () => {
+  const basePath = process.env.REACT_APP_DPOS_SUBGRAPH_HOST || '';
```

Comment:
Do we need to show some feedback when fetching the delegations and commission changes? If so, it might be good to add `const { startLoading, finishLoading } = useLoading();` to the top and use it when fetching the data.